.gitignore

```diff
@@ -1,23 +1,158 @@
+# editor/OS cache files
 .idea/
 .DS_Store
-__pycache__/
 .vscode/
+**/.DS_Store
 
 # jupyter notebooks
 .ipynb_checkpoints/
 
-# packaging
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
 dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
 *.egg-info/
-build/
+.installed.cfg
+*.egg
+MANIFEST
```

Comment:
Does this line exclude MANIFEST.in?

Reply:
No, it doesnt exclude

Reply:
can you still remove it? I think it's confusing since we have a MANIFEST.in file